tests/autoyast/login.pm

```diff
@@ -19,7 +19,9 @@
 use strict;
 use base 'y2logsstep';
 use testapi;
-use ipmi_backend_utils;
+# TODO: is_remote_backend we need to ensure if for autoyast we want this or
```

Comment:
Sorry, I don't understand that sentence.

@rwx788 any feedback?

Reply:
I'm not sure what's the motivation for that change. What I can acknowledge is that for autoyast we have some code which reinvents the wheel, which mainly is some legacy reasons to exist.

Reply:
@okurz What you don't understand about what sentence? (Unless you are talking about a TODO note that it's a TODO note?) :)

Reply:
I did not understand the english. I assume when "is_remote_backend" is not really start of the sentence I can understand it better. I guess the answer is: Yes